CHANGELOG.rst

```diff
@@ -1,6 +1,11 @@
 What’s new in django-cachalot?
 ==============================
 
+2.3.6
```

Comment:
Change minor version to 2.4.0 to signify the dropping of the Django and Python versions. This should also be mentioned in the CHANGELOG.
```suggestion
2.4.0
```